tt_eager/tt_dnn/op_library/conv/kernels/conv_bmm_tilize_col_major_out_blocks.cpp

```diff
@@ -33,25 +33,18 @@ inline void tilize_in(
     uint32_t in_num_subblocks,
     uint32_t out_cb_id) {
 
-    // Configure to RowMajor for tilize (similar to add bcast for bias)
-    MATH(( llk_math_pack_sync_init<SYNC>()  ));
-    PACK(( llk_pack_dest_init<SYNC, DstTileFaceLayout::RowMajor, false>()  ));
-
     tilize_init_short(in_cb_id, in_block_w);
     for (uint32_t in_subblock = 0; in_subblock < in_num_subblocks; ++in_subblock) {
         for (uint32_t h = 0; h < in_subblock_h; ++h) {
             cb_wait_front(in_cb_id, in_block_w);
```

Comment:
This could be the only change to compute , with an #ifdef 
 
      cb_wait_front(in_cb_id_top_half_of_tile, in_block_w);
       cb_wait_front(in_cb_id_botton_half_of_tile, in_block_w);
       
 
       cb_reserve_back(out_cb_id, in_block_w);
       tilize_block(in_cb_id, in_block_w, out_cb_id);
       cb_push_back(out_cb_id, in_block_w);

       cb_pop_front(in_cb_id_top_half_of_tile, in_block_w);
       cb_pop_front(in_cb_id_botton_half_of_tile, in_block_w);